python2/koans/about_monkey_patching.py

```diff
@@ -15,7 +15,7 @@ def bark(self):
 
```

Comment:
Are you familiar with the concept of Monkey Patching? Could you briefly explain what it is and give an example of what it is used for?

Reply:
I have an understanding of what is, but I'm not that familiar with it.

What I get of Monkey Patching is a way to modify classes to add functionalities according to what you desire for that specific code.

For example, you may be importing a class from another source, but for that specific code you need some kind of test to validate a variable, so instead of creating an object to handle that testing, you just add it to the imported class, and that would be valid for that code.